src/Microsoft.AspNet.Server.Kestrel/Networking/UvWriteReq2.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.AspNet.Server.Kestrel.Infrastructure;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Server.Kestrel.Networking
+{
+    /// <summary>
+    /// Summary description for UvWriteRequest2
+    /// </summary>
+    public class UvWrite2Req : UvRequest
```

Comment:
The class name and file name are inconsistent. I personally prefer `UvWriteReq2`.